### PR TITLE
how-to/maintain: fix the description of backup & restore document

### PR DESCRIPTION
**Pull request by GregoryIan**
### What is changed, added or deleted? 

the backup & restore document has some wrong descriptions

### Which version does your change affect? 
v3.0

### CODE REVIEW

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 1]
```diff
@@ -83,6 +83,43 @@ In this command,
 - `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
 - `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.
 
+If `mydumper` meeting error likes
+
+```log
+** (mydumper:27528): CRITICAL **: 13:25:09.081: Could not read data from samsung_bench.TEST_TBL: GC life time is shorter than transaction duration, transaction starts at 2019-08-05 21:10:01.451 +0800 CST, GC safe point is 2019-08-05 21:14:53.801 +0800 CST
+```
+
+two more command need to executed:
```
Comment (amyangfei) [inline v3.0/how-to/maintain/backup-and-restore.md]:
```suggestion
two more commands need to be executed:
```

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 2]
```diff
@@ -83,6 +83,43 @@ In this command,
 - `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
 - `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.
 
+If `mydumper` meeting error likes
+
+```log
+** (mydumper:27528): CRITICAL **: 13:25:09.081: Could not read data from samsung_bench.TEST_TBL: GC life time is shorter than transaction duration, transaction starts at 2019-08-05 21:10:01.451 +0800 CST, GC safe point is 2019-08-05 21:14:53.801 +0800 CST
```
Comment (amyangfei) [inline v3.0/how-to/maintain/backup-and-restore.md]: `samsung_bench.TEST_TBL` should we use another schema.table name, which is more like an example?

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 3]
```diff
@@ -83,6 +83,43 @@ In this command,
 - `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
 - `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.
 
+If `mydumper` meeting error likes
```
Comment (kennytm) [inline v3.0/how-to/maintain/backup-and-restore.md]:
```suggestion
If `mydumper` emits errors like
```

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 4]
```diff
@@ -83,6 +83,43 @@ In this command,
...
+  | tikv_gc_leader_lease  | 20180418-11:02:30 +0800 CST                                                                    |
+  | tikv_gc_run_interval  | 10m0s                                                                                          |
+  | tikv_gc_life_time     | 10m0s                                                                                          |
+  | tikv_gc_last_run_time | 20180418-10:59:30 +0800 CST                                                                    |
+  | tikv_gc_safe_point    | 20180418-10:58:30 +0800 CST                                                                    |
+  | tikv_gc_concurrency   | 1                                                                                              |
+  +-----------------------+------------------------------------------------------------------------------------------------+
+  10 rows in set (0.02 sec)
+
+  update mysql.tidb set VARIABLE_VALUE = '720h' where VARIABLE_NAME = 'tikv_gc_life_time';
```
Comment (kennytm) [inline v3.0/how-to/maintain/backup-and-restore.md]:
```suggestion
mysql> update mysql.tidb set VARIABLE_VALUE = '720h' where VARIABLE_NAME = 'tikv_gc_life_time';
```

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 5]
```diff
@@ -83,6 +83,43 @@ In this command,
...
+  ```
+
+- step 2:after finishing `mydumper` command, you need to restore the GC value of TiDB cluster to its origin value in step 1
```
Comment (kennytm) [inline v3.0/how-to/maintain/backup-and-restore.md]:
```suggestion
- step 2: after finishing the `mydumper` command, you need to restore the GC value of TiDB cluster to its original value in step 1
```

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 6]
```diff
@@ -83,6 +83,43 @@ In this command,
...
+```log
+** (mydumper:27528): CRITICAL **: 13:25:09.081: Could not read data from testSchema.testTable: GC life time is shorter than transaction duration, transaction starts at 2019-08-05 21:10:01.451 +0800 CST, GC safe point is 2019-08-05 21:14:53.801 +0800 CST
+```
+
+two more commands need to be executed:
+
+- step 1: before executing `mydumper` command, you need to query the GC values of TiDB cluster and adjust it to a suitable value using the MySQL client:
+
+  ```sql
+  mysql> select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb;
```
Comment (kennytm) [inline v3.0/how-to/maintain/backup-and-restore.md]: Perhaps use

```sql
SELECT * FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time'
```

to shorten the output below.

File: v3.0/how-to/maintain/backup-and-restore.md [hunk 7]
```diff
@@ -83,6 +83,34 @@ In this command,
 - `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
 - `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.
 
+If `mydumper` emits error likes
```
Comment (kennytm) [inline v3.0/how-to/maintain/backup-and-restore.md]:
```suggestion
If `mydumper` emits error like
```

The "like" here is a preposition, not a verb, there's no need to add the -s.

**Comment by GregoryIan** [comment]
thanks @lilin90

**Pull request merged**